backend/src/controllers/DailyWorkoutsController.js

```diff
@@ -24,7 +28,89 @@ const getDailyWorkoutsbyDate = async (req, res) => {
     }
 }
 
+
+  const insertWorkout = async (req, res) => {
+    const userId = req.user.data.user_id;
+    const { dailyworkout_id, workout_id, sets, reps } = req.body;
```

Comment:
`dailyworkout_id` isn't needed from the request body. As its a unique primary key since we are adding a new workout.